File: unittest/scripts/devapi/scripts/connecting_to_a_session.js
// Assumptions: test schema exists
// Passing the paramaters in the { param: value } format
var dictSession = mysqlx.getSession( {
        host: 'localhost', port: 33060,
        dbUser: 'mike', dbPassword: 's3cr3t!' } );
        
var db1 = dictSession.getSchema('test');

// Passing the paramaters in the URL format
var uriSession = mysqlx.getSession('mike:s3cr3t!@localhost:33060');

var db2 = uriSession.getSchema('test');

File: unittest/scripts/py_dev_api_examples/scripts/connecting_to_a_session.py
# Assumptions: test schema exists
# Passing the paramaters in the { param: value } format
dictSession = mysqlx.getSession( {
        'host': 'localhost', 'port': 33060,
        'dbUser': 'mike', 'dbPassword': 's3cr3t!' } )

db1 = dictSession.getSchema('test')

# Passing the paramaters in the URL format
uriSession = mysqlx.getSession('mike:s3cr3t!@localhost:33060')

db2 = uriSession.getSchema('test')